pandas/tests/arithmetic/test_period.py

```diff
@@ -755,18 +755,18 @@ def test_pi_sub_isub_offset(self):
         rng -= pd.offsets.MonthEnd(5)
         tm.assert_index_equal(rng, expected)
 
-    def test_pi_add_offset_n_gt1(self, box_transpose_fail):
+    @pytest.mark.parametrize("transpose", [True, False])
+    def test_pi_add_offset_n_gt1(self, box_with_array, transpose):
```

Comment:
not a blocker, but transpose param is sub-optimal.  For DataFrame case it will correctly test both cases, but for EA/Index/Series it will mean duplicate tests.  I'll try to come up with something nicer in an upcoming arithmetic-test-specific pass